test/test_jit.py

```diff
@@ -9440,6 +9439,7 @@ def func(x):
     ('softplus', (S, S, S), (),),
     ('softmin', (S, S, S), (0,),),
     ('softmax', (S, S, S), (0,),),
+    ('softmax', (S, S, S), (0, 3, torch.double), 'all_args'),
```

Comment:
What does 'all_args' do ? do you mind adding a comment above, next to where 'inplace' has a comment

Reply:
It doesn't do anything, it's just appended to the test name, changed to `with_all_args` for more clarity